xtask/src/main.rs

```diff
@@ -52,7 +55,8 @@ fn main() -> anyhow::Result<()> {
         }
         TestCommand::TestHost => test_host(opt.deny_warnings),
         TestCommand::TestCross => test_cross(),
-        TestCommand::TestSnapshot => test_snapshot(),
+        TestCommand::TestSnapshot { name: None } => test_snapshot(),
+        TestCommand::TestSnapshot { name: Some(name) } => test_single_snapshot(&name, "", false)?,
```

Comment:
Do we want to use `do_test(|| test_single_snapshot(test, features, false), "qemu/snapshot");` ? And do we want to only run in debug mode, or debug and release?

Reply:
Good question, I wanted to run a single snapshot program to minimize test time.